Mac/MainWindow/MainWindowController.swift

```diff
@@ -1069,6 +1070,15 @@ private extension MainWindowController {
 				}
 				return currentLink != nil
 			}
+			
+			if let webfeed = currentTimelineViewController?.selectedArticles.first?.webFeed {
+				if webfeed.isFeedProvider {
+					toolbarButton.isEnabled = false
+					return false
+				} else {
+					toolbarButton.isEnabled = true
+				}
+			}
```

Comment:
Is the toolbar item enabled or disabled when zero or multiple items are selected in the Timeline?

Reply:
The extractor is disabled in these scenarios.